iroh-ctl/src/main.rs

```diff
@@ -0,0 +1,32 @@
+mod status;
```

Comment:
if you create modules I would suggest making a `lib.rs` and declaring them there, makes for a cleaner organization